syft/formats/common/spdxhelpers/to_syft_model.go

```diff
@@ -280,24 +405,32 @@ func extractPkgInfo(p *spdx.Package) pkgInfo {
 	}
 }
 
-func toSyftPackage(p *spdx.Package) *pkg.Package {
+func toSyftPackage(p *spdx.Package) pkg.Package {
 	info := extractPkgInfo(p)
+	purl := ""
+	if isValidPURL(info.purl) {
+		purl = info.purl.String()
+	}
 	metadataType, metadata := extractMetadata(p, info)
-	sP := pkg.Package{
+	sP := &pkg.Package{
 		Type:         info.typ,
 		Name:         p.PackageName,
 		Version:      p.PackageVersion,
 		Licenses:     pkg.NewLicenseSet(parseSPDXLicenses(p)...),
 		CPEs:         extractCPEs(p),
-		PURL:         info.purl.String(),
+		PURL:         purl,
 		Language:     info.lang,
 		MetadataType: metadataType,
 		Metadata:     metadata,
 	}
 
 	sP.SetID()
 
-	return &sP
+	return *sP
+}
+
+func isValidPURL(purl packageurl.PackageURL) bool {
+	return purl.String() != "pkg:/"
```

Comment:
Hmm, is this an edge case? kind of curious.

Reply:
This is the default string if nothing is set on the purl struct, e.g.:
```
p := packageurl.PackageURL{}
p.String() // returns "pkg:/"
```